src/components/PassphraseInput.js

```diff
@@ -27,7 +27,10 @@ class PassphraseInput extends Nimiq.Observable {
 
         this.$input.placeholder = placeholder;
 
-        this.$eyeButton.addEventListener('click', () => this._changeVisibility());
+        this.$eyeButton.addEventListener('click', () => {
+            this._changeVisibility();
+            this.focus();
```

Comment:
Can we find out, if the input was focused before visibility was changed, to only re-focus when it had focus before? Or do you think it's expected to open the keyboard when I just click the visibility button on mobile?

Reply:
Functionality of the Eye Button is unchanged. I agree that we should talk about it.